src/interactions/autocompleters/ReminderAutocomplete.ts

```diff
@@ -0,0 +1,35 @@
+import { Autocompleter } from "../interactionClasses";
+import { AutocompleteInteraction } from "discord.js";
+import { reminderDb } from "../../db";
+import { timestampToReadable } from "../../util/reminderCommand/reminderUtil";
+
+class ReminderAutocompleter extends Autocompleter {
+	constructor() {
+		super("reminder");
+	}
+
+	async execute(interaction: AutocompleteInteraction): Promise<void> {
+		const options: { name: string; value: number }[] = [];
+
+		for (const [key, value] of reminderDb) {
+			if (value.pings[0] === interaction.user.toString()) {
+				const id = value.pings[1].replace(/\D/g, "");
+				const role = await interaction.guild?.roles.fetch(id);
+				const member = await interaction.guild?.members.fetch(id).catch(() => null);
```

Comment:
could we add the catch to the role fetch too

Reply:
not necessary 
the roles.fetch automatically returns null if the role with that id doesn't exist
discord.js is weird... check #help